futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -947,6 +950,67 @@ def validate_tenant_id(self, value: int) -> int:
             raise serializers.ValidationError('Unable to verify tenant access as request object is missing.')
 
         if value not in request.fx_permission_info['view_allowed_tenant_ids_full_access']:
-            raise serializers.ValidationError(f'User does not have have required access for teanant ({value}).')
+            raise serializers.ValidationError(f'User does not have have required access for tenant ({value}).')
 
         return value
+
+
+class TenantAssetSerializer(ModelSerializerOptionalFields):
+    """Serializer for Data Export Task"""
+    file_url = serializers.SerializerMethodField()
+    file = serializers.FileField(write_only=True)
+
+    class Meta:
+        model = TenantAsset
+        fields = ['id', 'tenant', 'slug', 'file', 'file_url', 'created_by', 'created_at']
```

Comment:
please use `tenant_id` for consistency with all other APIs
```suggestion
    tenant_id = serializers.PrimaryKeyRelatedField(queryset=TenantConfig.objects.all(), source='tenant')

    class Meta:
        model = TenantAsset
        fields = ['id', 'tenant_id', 'slug', 'file', 'file_url', 'created_by', 'created_at']
```